tests/unit/capi/GEOSLineMergeDirectedTest.cpp

```diff
@@ -0,0 +1,64 @@
+//
+// Test Suite for C-API GEOSLineMergeDirected
+
+#include <tut/tut.hpp>
+// geos
+#include <geos_c.h>
+// std
+#include <cstdarg>
+#include <cstdio>
+
+#include "capi_test_utils.h"
+
+namespace tut {
+//
+// Test Group
+//
+
+// Common data used in test cases.
+struct test_capigeoslinemergedirected_data : public capitest::utility {
+};
+
+typedef test_group<test_capigeoslinemergedirected_data> group;
+typedef group::object object;
+
+group test_capigeoslinemergedirected_group("capi::GEOSLineMergeDirected");
+
+//
+// Test Cases
+//
+
+template<>
+template<>
+void object::test<1>
+()
+{
+    auto input = GEOSGeomFromWKT("MULTILINESTRING((0 0, 0 100),(0 -5, 0 0))");
```

Comment:
If you use `geom1_` and `geom2_` as the variables, the `capitest::utility` will automatically clean them up after each test so you can drop the `GEOSGeom_destroy` calls.